operator/pkg/ingress/envoy_config.go

```diff
@@ -261,7 +261,7 @@ func getResources(ingress *slim_networkingv1.Ingress, backendServices []*v2.Serv
 
 func getListenerResource(ingress *slim_networkingv1.Ingress, secretNamespace string, enforcedHTTPS bool) (v2.XDSResource, error) {
```

Comment:
can you add a small unit tests to cover changes in this PR ? 

https://github.com/cilium/cilium/blob/master/operator/pkg/ingress/envoy_config_test.go#L49